### DISCUSSION TITLE
Chi with different widths

### DISCUSSION THREAD

**Discussion by Star-Ruby** [Part-II-Computational-Physics/.github#71] [Q&A]
My value for chi shrinks dramatically with higher lattice widths, what is the last task looking for when it's asking about why and how it varies with width?
![image](https://user-images.githubusercontent.com/90109705/224158082-f92a347b-6da8-4b02-a021-5437cfa36b00.png)

**Comment by DLHvdH** [comment]
Think about the core principles of this system. How do the spins interact? What happens to these interactions if you decrease the lattice size? Maybe reading one of the optional assignments will give you an idea.
